src/templates/search.html

```diff
@@ -164,6 +192,18 @@ <h2>{% trans "No results found" %}</h2>
             </form>
             </center>
         </div>
+        <br />
+        <br />
+        {% if faq != None %}
+            <div class="faq-box">
+                <h2>{% trans "Frequently Asked Questions" %}</h2>
```

Comment:
This looks a bit large on mobile devices — could we do `h3` instead?
And perhaps add a rule for
```
.faq-box h3 {
    margin: 13px;
}
```
so the top doesn't look weird.

Reply:
Changed to h3